spec/system/goobi_xml_imports_spec.rb

```diff
@@ -5,17 +5,20 @@
   let(:metadata_cloud_response_body_1) { File.open(File.join(fixture_path, "ladybird", "2012315.json")).read }
 
   before do
-    stub_request(:get, "https://metadata-api-test.library.yale.edu/metadatacloud/api/ladybird/oid/2012315")
+    stub_request(:get, "https://yul-development-samples.s3.amazonaws.com/ladybird/2012315.json")
       .to_return(status: 200, body: metadata_cloud_response_body_1)
+    prep_metadata_call
```

Comment:
do you need to do this if you are not calling the metadata api?

Reply:
It's needed in order to create a parent object